src/tmxrasterizer/tmxrasterizer.qbs

```diff
@@ -8,7 +8,7 @@ QtGuiApplication {
     Depends { name: "libtiled" }
 
     cpp.includePaths: ["."]
-    cpp.rpaths: ["$ORIGIN/../lib"]
+    cpp.rpaths: qbs.targetOS.contains("darwin") ? ["@loader_path/../Frameworks"] : ["$ORIGIN/../lib"]
```

Comment:
Since this depends on the install location of `libtiled`, does it make sense to move it to the `Export` part in `libtiled.qbs`? Or is this not possible / desired?